spec/models/like_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
```

Comment:
- [ ] All the methods in the `Like` Model should be tested, kindly make sure to implement unit tests for the `update_posts_like_counter` method